### ISSUE TITLE
Multiblock Madness AE2 Booster card crash 

### ISSUE THREAD

**Issue by DarksRoth** [PrototypeTrousers/Applied-Energistics-2#144]
**Describe the bug**

When i try to place ae2 booster cards in a wireless crafting terminal, the game crashes

**To Reproduce**

1) Take a wireless crafting terminal
2) Place a booster card in the slot

**Expected behavior**

Let me use the booster cards for more range

**Additional context**

https://pastebin.com/aTSPZE6K

**Environment**

Playing Multiblock Madness modpack

- Minecraft Version: 
- AE2 Version: appliedenergistics2-rv6-stable-7-extended_life-v0.52.10
- Forge Version: Forge - 14.23.5.2860

**Comment by Preta-Crowz** [comment]
> `OptiFine_1.12.2_HD_U_E3`
You are using outdated optifine. It might not issue with AE itself.
You may need to report it to WCT repo, after check it's not caused from optifine.

**Comment by TheAirBlow** [comment]
@DarksRoth this issue was fixed so you can close this issue now.

**Comment by Preta-Crowz** [comment]
> Bad idea to blame optifine for everything. The damn issue is literally `onSlotChanged` which is in the crash log and there's no way that optifine would ever touch these for any reason. Optifine is made to optimise the rendering and this is not related to it at all. Also what does WCT mean?
WCT is WirelessCraftingTerminal, it's not from AE itself and from own mod

**Comment by TheAirBlow** [comment]
> > Bad idea to blame optifine for everything. The damn issue is literally `onSlotChanged` which is in the crash log and there's no way that optifine would ever touch these for any reason. Optifine is made to optimise the rendering and this is not related to it at all. Also what does WCT mean?
> 
> WCT is WirelessCraftingTerminal, it's not from AE itself and from own mod

Indeed, but WCT can't be the problem here if it works in original AE2 but is broken for AE2EL
